src/style/index.style.scss

```diff
@@ -20,4 +20,8 @@
   margin        : 0;
   font-family   : $main-font;
   text-rendering: geometricPrecision;
+}
+
+.light {
+  background: slighten($color: $main-color, $amount: 20)
 }
```

Comment:
Might have been a typo here. Won't compile with the function `slighten`, but compiles with a function `lighten`.